tests/periph_flashpage/README.md

```diff
@@ -34,5 +34,5 @@ What else to check:
 
 Background
 ==========
-This test provides you with tools to test implementations of the `flashpage`
-peripheral driver interface.
+This test provides tools to test implementations of the `flashpage` peripheral
```

Comment:
> test ... to test

Seems somewhat redundant. :-)
What about "This shell provides commands to test ...."?

Reply:
Not exactly your suggestion, but I just pushed an update of this.